src/services/refactors/addOrRemoveBracesToArrowFunction.ts

```diff
@@ -55,12 +56,17 @@ namespace ts.refactor.addOrRemoveBracesToArrowFunction {
             body = needsParentheses(actualExpression) ? createParen(actualExpression) : actualExpression;
             suppressLeadingAndTrailingTrivia(body);
             copyLeadingComments(returnStatement, body, file, SyntaxKind.MultiLineCommentTrivia, /* hasTrailingNewLine */ false);
+            copyTrailingAsLeadingComments(returnStatement, body, file, SyntaxKind.MultiLineCommentTrivia, /* hasTrailingNewLine */ false);

```

Comment:
`copyTrailingAsLeadingComments` needs to come before `copyLeadingComments` or it will flip the order of the two.

Reply:
Ah whoops! Thanks for catching that.